### PR TITLE
Perl string util

### PR DESCRIPTION
**Pull request by Takadonet**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/perl-number-misc/1.2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  name: perl-number-misc
+  version: "1.2"
+
+source:
+  fn: Number-Misc-1.2.tar.gz
+  url: https://cpan.metacpan.org/authors/id/M/MI/MIKO/Number-Misc-1.2.tar.gz
+  md5: 7a0009b5be60378f204ed1fc9b2d5a19
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/perl-number-misc/1.2/meta.yaml]: build number should start with 0 for new packages.

File: recipes/perl-string-util/1.26/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package:
+  name: perl-string-util
+  version: "1.26"
+
+source:
+  fn: String-Util-1.26.tar.gz
+  url: https://cpan.metacpan.org/authors/id/M/MI/MIKO/String-Util-1.26.tar.gz
+  md5: d0f13f8688bedef7174634bb57f7fbe4
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/perl-string-util/1.26/meta.yaml]: Please start with 0.

File: recipes/perl-test-toolbox/0.4/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  name: perl-test-toolbox
+  version: "0.4"
+
+source:
+  fn: Test-Toolbox-0.4.tar.gz
+  url: https://cpan.metacpan.org/authors/id/M/MI/MIKO/Test-Toolbox-0.4.tar.gz
+  md5: b8a83d332dcd0691b68ed9e0a0a53e7b
+
+build:
+  number: 1
```
Comment (bgruening) [inline recipes/perl-test-toolbox/0.4/meta.yaml]: here as well

File: recipes/perl-number-misc/1.2/meta.yaml (hunk 1 above)
Comment (Takadonet) [inline recipes/perl-number-misc/1.2/meta.yaml]: Fixed.

File: recipes/perl-string-util/1.26/meta.yaml (hunk 2 above)
Comment (Takadonet) [inline recipes/perl-string-util/1.26/meta.yaml]: Fixed.

File: recipes/perl-test-toolbox/0.4/meta.yaml (hunk 3 above)
Comment (Takadonet) [inline recipes/perl-test-toolbox/0.4/meta.yaml]: Fixed.

**Comment by bgruening** [comment]
`Can't locate Module/Build.pm in @INC` you can depend on `perl-module-build` to resolve this.

**Comment by Takadonet** [comment]
Added perl-module-build and seems to pass now!

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**